### PR TITLE
fix(modal, www): Modal update  CL-103

### PR DESCRIPTION
**Pull request by mlipardo**
- [x] Update Documentation (Guidesite) with more examples

### CODE REVIEW

**Comment by eckdanny-osi** [comment]
We can safely ignore the 1 issue codeclimate identified (long render). The whole point of that example is how the component handles lengthy content 😃

File: apps/www/src/modules/components/articles/Modal/Modal.mdx [hunk 1]
```diff
@@ -1,21 +1,79 @@
...
+  ModalBody,
+  ModalFooter,
+  ModalHeader,
+  Button,
+  Icon,
+  IconButton,
+  Card,
+  CardBody
+} from "@cwds/components";
+import { CodeBlock, DemoCard, PropTable, Refere } from "@cwds/docs";
```
Comment (eckdanny-osi) [inline apps/www/src/modules/components/articles/Modal/Modal.mdx]: `Refere` ???

File: libs/reactstrap/src/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Modal } from 'reactstrap'
+
+/**
+ * @todo: use `scrollable` prop after reactstrap ships [#1435](https://github.com/reactstrap/reactstrap/pull/1435)
+ */
+Modal.defaultProps.modalClassName = 'modal-dialog-scrollable'
```
Comment (eckdanny-osi) [inline libs/reactstrap/src/Modal.jsx]: ☑️ a technical debt item was added to JIRA to track this.

File: apps/www/src/modules/components/articles/Modal/Modal.mdx (hunk 1 above)
Comment (eckdanny-osi) [reply apps/www/src/modules/components/articles/Modal/Modal.mdx]: 👍

**Comment by mlipardo** [comment]
retest this please

**Pull request merged**